cli.go

```diff
@@ -170,43 +175,28 @@ func (c *Cli) RunInteractive() int {
 }
 
 func (c *Cli) RunBatch(input string, displayTable bool) int {
-	// execute batched only if all statements are DDL statements
-	if stmts := buildDdlStatements(input); stmts != nil {
-		result, err := stmts.Execute(c.Session)
-		if err != nil {
-			c.PrintBatchError(err)
-			return exitCodeError
-		}
-
-		if displayTable {
-			c.PrintResult(result, DisplayModeTable, false)
-		} else {
-			c.PrintResult(result, DisplayModeTab, false)
-		}
-		return exitCodeSuccess
+	stmts, err := buildCommands(input)
```

Comment:
Thanks for renaming this! Then this should be `cmds, err := buildCommands(input)`?

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/40/commits/b8453c042c1ba8c7f7d1d56d212b2686a4d2e433